.github/workflows/dapr.yml

```diff
@@ -60,8 +60,13 @@ jobs:
         if: matrix.target_arch != 'arm' && matrix.target_os != 'windows'
         run: make lint
       - name: Run make test
+        env:	
+          COVERAGE_OPTS: "-coverprofile=coverage.txt -covermode=atomic"
         if: matrix.target_arch != 'arm'
         run: make test
+      - name: Codecov
+        if: matrix.target_arch != 'arm'
```

Comment:
I think we need code coverage only one time. e.g. linux amd64
so in this case, I would set `if` condition to `matrix.target_arch == 'amd64' && matrix.target_os == 'linux'`

Reply:
Done